azure-pipelines.yml

```diff
@@ -0,0 +1,32 @@
+# Docker image
+# Build a Docker image to deploy, run, or push to a container registry.
+# Add steps that use Docker Compose, tag images, push to a registry, run an image, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/docker
+
+trigger:
+- master
+
+pr:
+- master
+
+pool:
+  vmImage: 'Ubuntu-16.04'
+
+variables:
+- group: hwsc-dev-container-vars
+
+steps:
+- task: DownloadSecureFile@1
+  inputs:
+    secureFile: hwscdevcontainer_pw.txt
```

Comment:
did you change something in this file?

Reply:
yes i added variable `hwscDevImageNameUserSvc` with value `dev-hwsc-user-svc` in azure pipeline

Reply:
i am talking about this file `hwscdevcontainer_pw.txt`

Reply:
no, i was confused where this file even came from. i tried changing the name of it and it failed on me.